lighthouse-core/gather/gather-runner.js

```diff
@@ -100,7 +100,10 @@ class GatherRunner {
     const resetStorage = !options.flags.disableStorageReset;
     // Enable emulation based on flags
     return driver.assertNoSameOriginServiceWorkerClients(options.url)
-      .then(_ => driver.getUserAgent())
+      .then(_ => {
+        options.fetchedAt = (new Date()).toJSON();
```

Comment:
think we can stick it on gathererResults instead like we do for user agent?

Reply:
It's gonna be an artifact though? Replaced with `gatherRunner` and when running the tests ( 2 failed ), this is what came up:

<img width="558" alt="skjermbilde 2018-03-17 kl 00 48 26" src="https://user-images.githubusercontent.com/16735925/37549065-06df64b6-297d-11e8-8ff6-f5a0e1b9f2e2.png">

Lighthouse has `result.fetchedAt` as a prop above the artifact, but it resolves to undefined

Reply:
it makes a little more semantic sense to have it as an artifact like UserAgent as it is produced through the gathering process. also I think we'll have to to get this to work with the `lighthouse -G && lighthouse -A` workflow that paul flagged as part of #4753 since we don't save options to disk, just artifacts

Reply:
it might need some updating of tests as you noticed though :) sorry!

Reply:
Makes sense now as you say it, a bit busy now, but going to address it during the weekend 👍